modules/custom/az_core/src/AZConfigOverride.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Drupal\az_core;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Extension\ModuleExtensionList;
+use Drupal\config_provider\Plugin\ConfigCollector;
+use Drupal\az_core\Plugin\ConfigProvider\QuickstartConfigProvider;
+
+/**
+ * Class AZConfigOverrideIMport.
```

Comment:
This comment should be updated to provide better documentation of the override service.